### PR TITLE
New recipe for GBSX.

### PR DESCRIPTION
**Pull request by tesujimath**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by tesujimath** [comment]
@bioconda/core Review please!  :-)

File: recipes/gbsx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  host:
+  run:
+    - openjdk >=6
+    - python
+
+test:
+  commands:
+    - gbsx -help
+
+extra:
```
Comment (bgruening) [inline recipes/gbsx/meta.yaml]: can you add this doi:10.1186/s12859-015-0514-3 as an identifier?

Comment (tesujimath) [reply recipes/gbsx/meta.yaml]: @bgruening Done

**Comment by tesujimath** [comment]
@bioconda/core There is some urgency in getting this package released on BioConda.  In fact, it is needed for today (!).  Is there any reason it can't be merged now? Thanks!

**Comment by jerowe** [comment]
Looks great. Thanks!

**Comment by tesujimath** [comment]
@jerowe thanks so much for merging promptly!

**Pull request merged**